auth/src/utils/sendOtp.js

```diff
@@ -0,0 +1,19 @@
+import twilio from 'twilio';
+import dotenv from 'dotenv'
+
+dotenv.config({ path: 'src/.env' })

```

Comment:
Again, no need to config dotenv in each just in app.js or index.js after an app is instantiated